python/taichi/misc/gui.py

```diff
@@ -133,6 +133,14 @@ def rect(self, topleft, bottomright, radius=1, color=0xFFFFFF):
         self.line(c, d, radius, color)
         self.line(d, a, radius, color)
 
+    def text(self, context, pos, font_size=15, color=0xFFFFFF):
+        import taichi as ti
+        # TODO: refactor Canvas::text
+        font_size = float(font_size)
+        pos = ti.vec(*pos)
+        color = ti.vec(0, 255, 0, 0)
```

Comment:
Note that your canvas uses `f32` and `color` should be `(0, 1, 0, 0)`. You may also want to change it to a value dependent on your `color` argument. After changing 255 to 1 I get anti-aliased font:

![Screenshot from 2020-05-29 07-44-59](https://user-images.githubusercontent.com/6553256/83256427-6aeca180-a180-11ea-9115-8bd2ed832a4a.png)